specification/migrate/resource-manager/Microsoft.Migrate/stable/2019-05-01/migrate.json

```diff
@@ -0,0 +1,3748 @@
+          "type": "string",
+          "description": "Azure location in which project is created."
+        },
+        "tags": {
+          "type": "object",
+          "description": "Tags provided by Azure Tagging service."
+        },
+        "properties": {
+          "description": "Properties of the project.",
+          "$ref": "#/definitions/ProjectProperties"
+        }
+      }
+    },
+    "ProjectProperties": {
+      "type": "object",
+      "description": "Properties of a project.",
+      "properties": {
+        "createdTimestamp": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Time when this project was created. Date-Time represented in ISO-8601 format.",
+          "readOnly": true
+        },
+        "updatedTimestamp": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Time when this project was last updated. Date-Time represented in ISO-8601 format.",
+          "readOnly": true
+        },
+        "serviceEndpoint": {
+          "type": "string",
+          "description": "Endpoint at which the collector agent can call agent REST API.",
+          "readOnly": true
+        },
+        "assessmentSolutionId": {
+          "type": "string",
+          "description": "Assessment solution ARM id tracked by Microsoft.Migrate/migrateProjects."
+        },
+        "projectStatus": {
+          "type": "string",
+          "enum": [
+            "Active",
+            "Inactive"
+          ],
+          "description": "Assessment project status.",
+          "x-ms-enum": {
+            "name": "ProjectStatus",
+            "modelAsString": true
+          }
+        },
+        "customerWorkspaceId": {
+          "type": "string",
+          "description": "URL of service map workspace created by customer."
+        },
+        "customerWorkspaceLocation": {
+          "type": "string",
+          "description": "Location of service map workspace created by customer."
+        },
+        "numberOfGroups": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Number of groups created in the project.",
+          "readOnly": true
+        },
+        "numberOfMachines": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Number of machines in the project.",
+          "readOnly": true
+        },
+        "numberOfAssessments": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Number of assessments created in the project.",
+          "readOnly": true
+        },
+        "lastAssessmentTimestamp": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Time when last assessment was created. Date-Time represented in ISO-8601 format. This value will be null until assessment is created.",
+          "x-nullable": true,
+          "readOnly": true
+        },
+        "provisioningState": {
```

Comment:
Could user set this field?

Reply:
No user cannot set this. Making it read only.